test/DynamoCoreWpfTests/PackagePathTests.cs

```diff
@@ -181,6 +214,45 @@ public void AddPackagePathsTest()
             Assert.AreEqual(1,GetPreviewValue("07d62dd8-b2f3-40a8-a761-013d93300444"));
         }
 
+        [Test]
+        public void InstalledPackagesContainsCorrectNumberOfPackages()
+        {
+            var pathManager = new Mock<IPathManager>();
+            pathManager.SetupGet(x => x.PackagesDirectories).Returns(
+                () => new List<string> { PackagesDirectory });
+
+            var loader = new PackageLoader(pathManager.Object);
+            var libraryLoader = new ExtensionLibraryLoader(ViewModel.Model);
+
+            loader.PackagesLoaded += libraryLoader.LoadPackages;
+            loader.RequestLoadNodeLibrary += libraryLoader.LoadLibraryAndSuppressZTSearchImport;
+
+            var packagesLoaded = false;
+
+            Action<IEnumerable<Assembly>> pkgsLoadedDelegate = (x) => { packagesLoaded = true; };
+            loader.PackagesLoaded += pkgsLoadedDelegate;
+
+            ViewModel.Model.PreferenceSettings.CustomPackageFolders = new List<string>();
+            var loadPackageParams = new LoadPackageParams
+            {
+                Preferences = ViewModel.Model.PreferenceSettings,
+
+            };
+            loader.LoadAll(loadPackageParams);
+            Assert.AreEqual(18, loader.LocalPackages.Count());
+            Assert.AreEqual(true, packagesLoaded);
+
+            var installedPackagesViewModel = new InstalledPackagesViewModel(ViewModel, loader);
+            Assert.AreEqual(18, installedPackagesViewModel.LocalPackages.Count);
+
+            var installedPackagesView = new Dynamo.Wpf.Controls.InstalledPackagesControl();
+            installedPackagesView.DataContext = installedPackagesViewModel;
```

Comment:
I couldn't actually test the absence of the `LocalPackages `property in `PreferencesViewModel `as this test does not strictly emulate the UI nor could I figure out a way to mock a `PreferencesViewModel `object instantiated with this `installedPackagesViewModel` used in this test. I guess a UI automation test is best for that.

Reply:
@mjkkirschner, @sm6srw, this test is actually not of much use to test this regression. I did some cleanup to the xaml. Please review one last time.

Reply:
I'm fine with that - if you do want to create a view and interact with it here is an exmaple:
https://github.com/DynamoDS/Dynamo/pull/11867/